spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -116,66 +63,54 @@ def self.spaceship_session_env
       ENV["FASTLANE_SESSION"] || ENV["SPACESHIP_SESSION"]
     end
 
-    def select_device(r, device_id)
+    def select_device(scnt, device_id)
       # Request Token
-      r = request(:put) do |req|
-        req.url "https://idmsa.apple.com/appleauth/auth/verify/device/#{device_id}/securitycode"
-        update_request_headers(req)
+      response = request(:post) do |req|
+        req.url "https://idmsa.apple.com/IDMSWebAuth/generateSecurityCode"
+        req.body = "deviceIndex=#{device_id}&scnt=#{scnt}"
```

Comment:
@taquitos Should we uri encode `device_id` and `scnt` as well? I don't know _exactly_ what these values look like but better safe than sorry if we don't have full control over this input?